apps/indexer-coordinator/src/project/rpc.factory.ts

```diff
@@ -144,7 +144,7 @@ export class RpcFamilyEvm extends RpcFamily {
     this.actions.push(async () => {
       const result = await jsonRpcRequest(this.endpoint, 'eth_getBalance', [
         '0x0000000000000000000000000000000000000000',
-        'latest',
+        '0x1',
```

Comment:
this may not be enough. I remember for erigon full node, it will return the getBalance request but with value of latest block

Reply:
or we can use more real request in the availability reporter. like use a real account, not only check the return code but also compare the return value.